src/handlers/shows.js

```
@@ -37,9 +42,12 @@ export async function handleFindShowIntent(req, resp) {
 }
 
 export async function handleAddShowIntent(req, resp) {
-  const api = getProvider(PROVIDER_TYPE.SHOWS);
   const showName = req.slot('showName');
+  if (!showName) {
+    return Promise.resolve(resp.say(NO_SHOW_SLOT()));
```

This should throw an error. Check out the movies.js file